p2panda-encryption/src/data_scheme/mod.rs

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! Data encryption for groups with post-compromise security and optional forward secrecy.
+//!
+//! This "Data Encryption" scheme allows peers to encrypt any data with a secret, symmetric key for
+//! a group. This will be useful for building applications where users who enter a group late will
+//! still have access to previously created content, for example private knowledge or wiki
+//! applications or a booking tool for rehearsal rooms.
+//!
+//! A member will not learn about any newly created data after removing them from the group since
```

Comment:
I've suggested some changes in the `README` for this section which could be copy-pasted here (I'll just mention that instead of suggesting all the same changes).